features/steps/steps.py

```diff
@@ -0,0 +1,358 @@
+from behave import *
```

Comment:
This file is where the code for the actual tests is included. I won't go into detail, but please peruse what's there so far, see if it makes sense.